### DISCUSSION TITLE
Feedback

### DISCUSSION THREAD

**Discussion by GlaDal** [opengisch/QField#3873] [Ideas]
Hi, 
1 - Could be nice having the possibility to check all the entities in the forms (after having displayed all the entities wich appeared in the map canvas from the menu layer for example). For now, we have to activatei the selection of entity but after that we must check all entities manually.

2 - Could be nice having a scroll bar when we use a list of values in the form. Actually, there is no visual element which inform that there are more than 5 elements in the list.

**Comment by lucienicolier** [comment]
Hi,

1. you are already able to check all change and choose if you applied or not in QFieldCloud, see the doc here [changes](https://docs.qfield.org/reference/qfieldcloud/projects/?h=changes#changes)

2. If you are interested to sponsor this feature, we will be really happy. Have a look in here [sponsor a feature](https://docs.qfield.org/get-started/sponsor/)
